src/main/java/duke/Output.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+
+public class Output {
+
+    public static void printStart() {
+        String logo = " ____        _        \n"
```

Comment:
Consider refactoring this into a constant instead of using a magic literal?

https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/refactoring.html